File: PYTHON/Gemini_1.5_Flash/Bridge/Round1/main.py
class Shape:
    def __init__(self, color):
        self.color = color

    def draw(self):
        pass

class Circle(Shape):
    def draw(self):
        print(f"Drawing a circle in {self.color} color")

class Square(Shape):
    def draw(self):
        print(f"Drawing a square in {self.color} color")

# สร้างออบเจ็กต์ของ Circle และ Square
circle = Circle("red")
square = Square("blue")

# เรียกใช้ฟังก์ชัน draw
circle.draw()  # จะแสดงผล: Drawing a circle in red color
square.draw()  # จะแสดงผล: Drawing a square in blue color

File: JAVA/gemini_1.5_flash/Bridge/round1/Main.java
package com.example;

public class Main {
    public static void main(String[] args) {
        // Create a red circle
        Shape redCircle = new Circle(new RedColor());
        redCircle.draw();  // Output: Drawing Circle with color. Coloring with Red.

        // Create a blue circle
        Shape blueCircle = new Circle(new BlueColor());
        blueCircle.draw();  // Output: Drawing Circle with color. Coloring with Blue.
    }
}
